### ISSUE TITLE
`Lint/ElseLayout` does not warn when else contains only one line code

### ISSUE THREAD

**Issue by ivodimitrov** [rubocop/rubocop#10130]
## Expected behavior

Warn when having an expression on the same line as the `else` keyword.

## Actual behavior

Does not warn when having an expression on the same line as the `else` keyword and else statement contains only one line code.

## Steps to reproduce the problem
Write one line code on the same line with else statement. Example:

```
if something
  # ...
else do_this
end
```

## RuboCop version

```
$ [bundle exec] rubocop -V
1.21.0 (using Parser 3.0.2.0, rubocop-ast 1.11.0, running on ruby 2.7.2 x86_64-darwin19)
```

**Comment by dvandersluis** [comment]
`Layout/ElseLayout` is only concerned with the actual alignment of the `else` and `elsif` keywords, not for not having other code on the same line.

That being said though, we probably should have a `Layout` cop for making sure the else body is not on the same line as the keyword.
